lambda_functions/healthcheck/healthcheck.py

```diff
@@ -51,12 +51,12 @@ def check_web_url_health(url, expected_string, env):
     # If the response code is not 200 or the response text does not 
     # contain the expected string then we log an error and fail the lambda
     if response.status_code != 200:
-        logger.info(f"Metric Data: , {metric_data}")
-        raise logger.error(f"Error: Status code expected to be 200 but is {response.status_code}")
+        print(f"Metric Data: , {metric_data}")
+        raise Exception(f"Error: Status code expected to be 200 but is {response.status_code}")
```

Comment:
The question is do you need this to raise an exception or do you actually just want to flag an error log? Does the alarm only trigger if exceptions are raised?

Reply:
we need to raise an error for the lambda to fail as that triggers the alarm

Reply:
so yes the alarm only triggers when exceptions are raised

Reply:
i have changed it to runtime error as exception does not fail lambda :)